src/client/entity-editor/entity-editor.tsx

```diff
@@ -81,13 +81,13 @@ const EntityEditor = (props: Props) => {
 					<AliasEditor show={aliasEditorVisible} {...props}/>
 					<NameSection {...props}/>
 					<ButtonBar {...props}/>
-					<RelationshipSection {...props}/>
 					{
 						React.cloneElement(
 							React.Children.only(children),
 							{...props}
 						)
 					}
+					<RelationshipSection {...props}/>
```

Comment:
props.children renders the EntitySection (WorkSection, AuthorSection, SeriesSection etc).
Untill now it was fine to have EntitySection below RelationshipSection but after Implementing SeriesSection it was very odd to have SeriesSection below RelationshipSection because it have some fields(seriesType, orderingType) which are needed to be filled before moving to RelationshipSection.

![Screenshot from 2021-06-23 09-02-07](https://user-images.githubusercontent.com/55311336/123031352-ccbe2000-d401-11eb-9d63-32dde1139f22.png)

Reply:
Makes sense, thanks.